web/themes/custom/vote_gov/src/js/external-links.js

```diff
@@ -2,38 +2,23 @@
   Processing External Links
  */
 
-  (() => {
+(() => {
 
-    // Enable external links behavior.
-    const externalLinks = document.querySelectorAll('a[href^="http"]');
-    // Set all whitelisted domains.
-    const whitelistedDomains = [
-      'https://www.gsa.gov',
-      'https://www.gsaig.gov',
-      'https://www.usa.gov'
-    ];
+  // Enable external links behavior.
+  const externalLinks = document.querySelectorAll('a[href^="https"]');
 
-    // Checks if a value matches one of the whitelistedDomains.
-    const matchesWhitelistedDomain = (value) => {
-      for (let i = 0; i < whitelistedDomains.length; i++) {
-        if (value.indexOf(whitelistedDomains[i]) != -1) {
-          return true;
-        }
-      }
+  const title_text = drupalSettings.title_text;
+  const title_translated = Drupal.t(title_text);
```

Comment:
remove title_text

Update
`const title_translated = Drupal.t('External link opens in new window');`